srcpkgs/qodem/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'qodem'
+pkgname=qodem
+version=1.0.1
+revision=1
+build_style=gnu-configure
+makedepends="ncurses-devel gpm-devel miniupnpc-devel SDL-devel"
+short_desc="Re-implementation of the Qmodem serial communication package"
+maintainer="Dakota Richline <drichline@protonmail.com>"
+license="CC0-1.0"
+homepage=https://qodem.sourceforge.net/
+distfiles="${SOURCEFORGE_SITE}/qodem/qodem-${version}.tar.gz"
+checksum=dedc73bfa73ced5c6193f1baf1ffe91f7accaad55a749240db326cebb9323359
+
+post_install() {
+	vman docs/qodem.1
```

Comment:
when I built it locally, it installed this file already

Reply:
In the future, I'll verify whether docs/licenses are installed by `configure` before doing it manually. Thank you again!!

Reply:
what I like to do is `xbps-src install pkgname` (which runs everything up to `post_install()`, then look at what's in `masterdir/destdir/pkgname-version/` (`tree` is useful for this), then `xbps-src pkg pkgname` to finish building

alternatively, there's `xls pkgname` from the `xtools` package if you want to list files from the finished result